modules/ROOT/pages/connected-apps-devs-end-users.adoc

```diff
@@ -5,12 +5,10 @@ endif::[]
 
 == Developers
 
-Third-party developers with their own websites or web apps can allow users to log in using the *Log in with Anypoint Platform* option. To make things easy, we crated a standard button that can be integrated to your application front end with via pre-built template and set of components. For more information about how to embed the button, refer to https://github.com/anypoint-web-components/anypoint-signin/tree/master[developer tutorial] and http://anypoint-signin-test.surge.sh/[developer playground]. 
+Third-party developers who have their own websites or web apps can allow users to log in using the *Log in with Anypoint Platform* option. For seamless implementation, there is a standard button that can be integrated into the front end of your application with a pre-built template and set of components. For more information about how to embed the button, refer to the  https://github.com/anypoint-web-components/anypoint-signin/tree/master[developer tutorial] and the http://anypoint-signin-test.surge.sh/[developer playground]. 
 
-In addition to enabling new flows to authenticate users, developers can enable or disable the clients, and view client usage.
-For example, they can see how many organizations are using the app and how many users have access.
-
-// code/chunk of steps for websites/web apps
+In addition to enabling new flows to authenticate users, developers can enable or disable the clients and view client usage.
```

Comment:
Suggest "clients" instead of "the clients"

Reply:
```suggestion
In addition to enabling new flows to authenticate users, developers can enable or disable clients and view client usage.
```